cmd/cluster/pitr-config/pitr_config_ops.go

```diff
@@ -97,15 +149,35 @@ var createPitrConfigCmd = &cobra.Command{
 			logrus.Debugf("Full HTTP response: %v", r)
 			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
 		}
+		pitrConfigId := resp.Data.Info.Id
 
-		fmt.Printf("\nSuccessfully created PITR configuration.\n\n")
+		msg := fmt.Sprintf("The PITR Configuration for namespace %s in cluster %s is being created\n\n", formatter.Colorize(namespaceName, formatter.GREEN_COLOR), formatter.Colorize(ClusterName, formatter.GREEN_COLOR))
 
-		pitrConfigCtx := formatter.Context{
-			Output: os.Stdout,
-			Format: formatter.NewPitrConfigFormat(viper.GetString("output")),
-		}
+		if viper.GetBool("wait") {
+			returnStatus, err := authApi.WaitForTaskCompletion(clusterID, ybmclient.ENTITYTYPEENUM_CLUSTER, ybmclient.TASKTYPEENUM_ENABLE_DB_PITR, []string{"FAILED", "SUCCEEDED"}, msg)
+			if err != nil {
```

Comment:
The error messages could be made more consistent. For example, in this command, you use `logrus.Fatalf("error when getting task status: %s", err)`, but in the delete command, it's `logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))`.

Reply:
The second error that you mentioned is from the create/delete/restore api calls which is consistent. The string error is from listTasks API calls